openstudiocore/src/model/SubSurface.cpp

```diff
@@ -735,6 +737,27 @@ namespace detail {
     }
   }
 
+  boost::optional<SurfacePropertyConvectionCoefficients> SubSurface_Impl::surfacePropertyConvectionCoefficients() const {
+    std::vector<SurfacePropertyConvectionCoefficients> allspccs(model().getConcreteModelObjects<SurfacePropertyConvectionCoefficients>());
+    std::vector<SurfacePropertyConvectionCoefficients> spccs;
```

Comment:
@macumber I haven't written any custom clone methods. I was planning on it, but ran out of time. Is it too late to get this in?